python/src/aiconfig/scripts/rage/rage.py

```diff
@@ -0,0 +1,60 @@
+import os
+import lastmile_utils.lib.core.api as core_utils
+import sys
+import time
+import itertools
+import numpy as np
+from result import Ok, Result
+
+logger = core_utils.get_logger(__name__)
+
+
+class RageConfig(core_utils.Record):
+    log_level: str | int = "WARNING"
+
+
+def rage(config: RageConfig) -> Result[None, str]:
+    logger.setLevel(config.log_level)
+    print("\n\n\n\n............\n\n")
+    print("Please open an issue! :) Here are some tips on how to do that:")
+    for logfile in ["editor_flask_server.log", "aiconfig.log"]:
+        print()
+        if os.path.exists(logfile):
+            print(f"Found {logfile}! Please include its contents in your bug report.")
+            print(f"Full path: {os.path.abspath(logfile)}")
+        else:
+            print(f"No {logfile} found. This might be another bug :)")
+            print("For now, please include your terminal output in your bug report.")
+
+    print("\nPlease run the following commands and also include their output:")
+    print("\nwhich pip; which pip3; which python; which python3; pip3 list | grep aiconfig; python --version; python3 --version")
+
+    print("\nPlease open an issue here: https://github.com/lastmile-ai/aiconfig/issues/")
+    print("Our sincerest gratitude. We will comment on the issue as soon as possible.")
+    print("\n\n")
+    print("Done raging! :)")
+    print("\n\n")
+
+    return Ok(None)
+
+
+def spin(seconds: int, type: str = "spinner"):
+    """This is just for fun."""
+
+    assert type in ["spinner", "music"]
+
+    spinning = itertools.cycle(["-", "/", "|", "\\"])
+
+    def get_animation():
+        if type == "spinner":
+            return next(spinning)
+        else:
+            return "".join(np.random.choice(["♩", "♫", "♬", "♪"]) for _ in range(5))
```

Comment:
Let's not use `np` if we can avoid and just use regular `from Random import random`. I get that it's in C so slightly faster, but this isn't needed

Reply:
Also lol this is pretty not-necessary at all, but it would be cute if we kept playing musical notes as we wait:

```
import numpy as np
from time import sleep

music_notes = [np.random.choice(["♩", "♫", "♬", "♪"]) for _ in range(50)]
# print(f"{music_notes=}")

for note in music_notes:
    sleep(0.5)
    print(note, end="")
```

https://github.com/lastmile-ai/aiconfig/assets/151060367/bd629e63-6d92-4b4f-a8aa-b5495dd9be86

Reply:
@rossdanlm , I get that numpy is heavier than we need here but it's very handy and easy to use - it's not about efficiency in this case. Let's not block on that